uint16 navboard_seq
uint16[3] acc
int16[3] gyro
int16[2] gyro_110
uint16 acc_temp
uint16 gyro_temp
uint16 vrefEpson
uint16 vrefIDG
uint16 us_echo_start
uint16 us_echo_end
uint16 us_association_echo
uint16 us_distance_echo
uint16 us_courbe_temps
uint16 us_courbe_valeur
uint16 us_courbe_ref
uint16[7] newUint16


